app/exec/extension/_lib/vsix-manifest-builder.ts

```diff
@@ -267,6 +267,30 @@ export class VsixManifestBuilder extends ManifestBuilder {
 					});
 				}
 				break;
+			case "repository":
+				if (_.isObject(value)) {					
+					let type = null, url = null;
+					Object.keys(value).forEach((key) => {
+						if( key.toLowerCase() === "type" && value[key].toLowerCase() === "git") {

```

Comment:
Please use consistent whitespace with the rest of this file (1 space after if keyword, no space between parenthesis and expr)